### PR TITLE
 #162235868 Create view endpoints for red flags

### PR DESCRIPTION
**Pull request by Curti-s**
### What does this PR do?
It ensures that there are view endpoints for red flags 

### Description of the completed task
Created the following endpoints:
- POST request for new red flag creation
- GET request for a particular red flag
- GET request for all red flags
- PUT request for a particular red flag
- DELETE request for a particular red flag

### How can this be manually tested
Source the .env file and run the flask app [flask run]
Define a collection in postman and perform GET POST PUT and DELETE request operations

### Screenshots
screenshot from 2018-11-28 11-30-00
screenshot from 2018-11-28 11-30-31
screenshot from 2018-11-28 11-30-43

### Relevant pivotal tracker stories
 #162235868

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
-from flask import Flask
+from flask import Flask, Blueprint
 from flask_restful import Resource, Api
 
 # local import
 from instance.config import app_config
+from .api.v1.views import redflag as rf
```
Comment (thoth-ky) [inline app/__init__.py:6]: Names should be meaningful to aid in readability and documentation of code
```suggestion
from .api.v1.views import redflag
```

File: app/api/v1/views/redflag_view.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+]
+
+class RedFlags(Resource):
+    """Redflags, lists all red flags and lets you create new ones"""
+    
+    def __init__(self):
+        self.db = redflag_list
+    
+    # create a red flag
+    def post(self):
```
Comment (thoth-ky) [inline app/api/v1/views/redflag_view.py:31]: This method does not seem to be using payload from the request.

**Review by thoth-ky — CHANGES_REQUESTED** [review]
Kindly work on the feedback I have given you. Use data provided by user, do not hardcode. Also ensure you write tests for this feature

File: app/api/v1/views/redflag_view.py (hunk 2 above)
Comment (Curti-s) [reply app/api/v1/views/redflag_view.py:31]: `Sure. I will refactor. Thanks for the feedback

**Pull request merged**